defaults/main.yml

```diff
@@ -34,21 +34,44 @@ cartridge_app_group: tarantool
 
 cartridge_custom_steps: []
 cartridge_custom_steps_dir: null
-cartridge_scenario:
-  - deliver_package
-  - update_package
-  - update_instance
-  - configure_instance
-  - restart_instance
-  - wait_instance_started
-  - connect_to_membership
-  - edit_topology
-  - cleanup_expelled
-  - configure_auth
-  - configure_app_config
-  - bootstrap_vshard
-  - configure_failover
-  - cleanup
+cartridge_scenario: null
+
+cartridge_scenario_name: 'default'
+cartridge_custom_scenarios: {}
+cartridge_role_scenarios:
+  default:
+    - deliver_package
+    - update_package
+    - update_instance
+    - configure_instance
+    - restart_instance
+    - wait_instance_started
+    - connect_to_membership
+    - edit_topology
+    - cleanup_expelled
+    - configure_auth
+    - configure_app_config
+    - bootstrap_vshard
+    - configure_failover
+  configure_instances:
+    - deliver_package
+    - update_package
+    - update_instance
+    - configure_instance
+    - restart_instance
+    - wait_instance_started
+    - cleanup
+  configure_topology:
+    - connect_to_membership
+    - edit_topology
+    - cleanup_expelled
+    - cleanup
+  configure_app:
+    - configure_auth
+    - configure_app_config
+    - bootstrap_vshard
+    - configure_failover
+    - cleanup
```

Comment:
I think that `cleanup` isn't a part of `configure_app`.

Reply:
I think that a cleanup should be in the end of each scenario, because user can replace any task and specify files to remove by `cleanup` task